switchmap/server/db/ingest/update/zone.py

```
@@ -262,16 +261,21 @@ def ip(self):
-                    hostname_map[ip] = socket.gethostbyaddr(ip)[0]
+                    hostname_map[item] = socket.gethostbyaddr(item)[0]
                 except Exception as e:
                     log.log2debug(
                         1035,
-                        f"Unexpected error during DNS lookup for {ip}: ",
-                        f"{str(e)}",
+                        f"""\
+Unexpected error during DNS lookup for {item}: {str(e)}""",
+                    )
+                    hostname_map[item] = None
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)